scripts/globals/gear_sets.lua

```diff
@@ -80,6 +80,7 @@ local GearSets =  {
              {id = 49, items = {18761,18597,17757,19218,18128,18500,16004,18951}, matches = 2, matchType = matchtype.earring_weapon, mods = {{MOD_STR, 6, 0, 0}, {MOD_ATT, 4, 0, 0}, {MOD_RATT, 4, 0, 0}, {MOD_MATT, 2, 0, 0}} }, --  Supremacy Earring Sets. Set Bonus: STR+6, Attack+4, Ranged Attack+4, "Magic Atk. Bonus"+2. Active with any 2 items(Earring+Weapon)
              
              {id = 53, items = {16006,18450,18499,18861,18862,18952,19111,19217,19272}, matches = 2, matchType = matchtype.earring_weapon, mods = {{MOD_EVA, 10, 0, 0}, {MOD_HPHEAL, 10, 0, 0}, {MOD_ENMITY, -5, 0, 0}} }, --  Brilliant Earring Set. Set Bonus: Evasion, HP Recovered while healing, Reduces Emnity. Active with any 2 items(Earring+Weapon)
+             {id = 56, items = {11798,11362}, matches = 2, matchType = matchType.any, mods = {{MOD_RERAISE_III, 1, 0, 0}} },        -- Twilight Mail Set. Set Bonus: Auto-Reraise

```

Comment:
should be ```matchType = matchtype.any```
or just replace all 'matchtype' with something else cause i lack the imagination to find a suitable name